src/universal/modules/outcomeCard/components/OutcomeCardAssignMenuRoot.js

```diff
@@ -18,41 +17,41 @@ const query = graphql`
   }
 `;
 
+const loading = (props) => <LoadableLoading {...props} height={DEFAULT_MENU_HEIGHT} width={DEFAULT_MENU_WIDTH} />;
+
+const LoadableOutcomeCardAssignMenu = Loadable({
+  loader: () => System.import(
+    /* webpackChunkName: 'OutcomeCardAssignMenu' */
+    'universal/modules/outcomeCard/components/OutcomeCardAssignMenu/OutcomeCardAssignMenu'
```

Comment:
testing the react-loadable pattern. per-component async loading is much cleaner than per-route.